packages/ContentsGroupingExtensions/src/components/ChildRelationField/index.vue

```diff
@@ -1,32 +1,49 @@
 <template>
     <div>
-        <dt class="after_second">{{ $attrs.ext_title }}</dt>
-        <dd
-            class="multi_dd"
-            :class="`ext_item_${$attrs.ext_index - 1}`"
-            :data-ext_type="$attrs.ext_type"
-            :data-default_value="$attrs.default_value"
-        >
-            <dl>
-                <ExtRelationFld
-                    v-bind="{
-                        ...$attrs,
-                        opts: $attrs.options,
-                    }"
-                />
-                <p v-if="$attrs.ext_help_msg">
-                    <span class="hint">{{ $attrs.ext_help_msg }}</span>
-                </p>
-            </dl>
-        </dd>
+        <div v-if="flagToLoadOnce">
+            <ExtRelationFld
+                v-bind="{
+                    call_from_module: 'topics',
+                    setting_id: $attrs.topics_group_id,
+                    ext_no: $attrs.name,
+                    selected_item: $attrs.value,
+                    table_id: $attrs.options.module,
+                    allowed_id: $attrs.options.group_id,
+                    limit: $attrs.options.limit,
+                    input_name: $attrs.name,
+                    disp_group_no: $attrs.options.disp_group_no,
+                    ext_options: $attrs.options,
+                }"
+            />
+        </div>
     </div>
 </template>
 
 <script>
 export default {
     inheritAttrs: false,
-    components: {
-        ExtRelationFld: () => import('./ExtRelationFld/index.vue'),
+    data() {
+        return {
+            flagToLoadOnce: false,
+        };
+    },
+    computed: {
+        isLoaded() {
+            // isLoaded in ContentsGroupingExtension
+            return this.$parent.$parent.isLoaded;
```

Comment:
I think it's not good checking parent components.
Please consider just passing prop instead.

Reply:
fixed